docs/deletion.md

```diff
@@ -0,0 +1,90 @@
+# Deletion of images conforming to DSGVO
```

Comment:
For all of these cases: should we add a sentence explaining that an old image gets deleted when it's a replaced by a new one (and not just when e.g. an idea is deleted) ?

Reply:
also one of the tests is failing :)